protobuf/python-types/mempool_pb2.py

```diff
@@ -3,31 +3,361 @@
-from google.protobuf import descriptor_pool as _descriptor_pool
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
 from google.protobuf import symbol_database as _symbol_database
-from google.protobuf.internal import builder as _builder
 # @@protoc_insertion_point(imports)
 
 _sym_db = _symbol_database.Default()
 
 
 
 
-DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\n\rmempool.proto\x12\x07mempool\"\xab\x01\n\x0cMempoolEvent\x12\x1f\n\x05\x61\x64\x64\x65\x64\x18\x01 \x01(\x0b\x32\x0e.mempool.AddedH\x00\x12%\n\x08replaced\x18\x02 \x01(\x0b\x32\x11.mempool.ReplacedH\x00\x12#\n\x07removed\x18\x03 \x01(\x0b\x32\x10.mempool.RemovedH\x00\x12%\n\x08rejected\x18\x04 \x01(\x0b\x32\x11.mempool.RejectedH\x00\x42\x07\n\x05\x65vent\"1\n\x05\x41\x64\x64\x65\x64\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\r\n\x05vsize\x18\x02 \x02(\x05\x12\x0b\n\x03\x66\x65\x65\x18\x03 \x02(\x03\"W\n\x07Removed\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\x0e\n\x06reason\x18\x02 \x02(\t\x12\r\n\x05vsize\x18\x03 \x02(\x05\x12\x0b\n\x03\x66\x65\x65\x18\x04 \x02(\x03\x12\x12\n\nentry_time\x18\x05 \x02(\x04\"(\n\x08Rejected\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\x0e\n\x06reason\x18\x02 \x02(\t\"\xba\x01\n\x08Replaced\x12\x15\n\rreplaced_txid\x18\x01 \x02(\x0c\x12\x16\n\x0ereplaced_vsize\x18\x02 \x02(\x05\x12\x14\n\x0creplaced_fee\x18\x03 \x02(\x03\x12\x1b\n\x13replaced_entry_time\x18\x04 \x02(\x04\x12\x18\n\x10replacement_txid\x18\x05 \x02(\x0c\x12\x19\n\x11replacement_vsize\x18\x06 \x02(\x05\x12\x17\n\x0freplacement_fee\x18\x07 \x02(\x03')
+DESCRIPTOR = _descriptor.FileDescriptor(
+  name='mempool.proto',
+  package='mempool',
+  syntax='proto2',
+  serialized_options=None,
+  create_key=_descriptor._internal_create_key,
+  serialized_pb=b'\n\rmempool.proto\x12\x07mempool\"\xab\x01\n\x0cMempoolEvent\x12\x1f\n\x05\x61\x64\x64\x65\x64\x18\x01 \x01(\x0b\x32\x0e.mempool.AddedH\x00\x12%\n\x08replaced\x18\x02 \x01(\x0b\x32\x11.mempool.ReplacedH\x00\x12#\n\x07removed\x18\x03 \x01(\x0b\x32\x10.mempool.RemovedH\x00\x12%\n\x08rejected\x18\x04 \x01(\x0b\x32\x11.mempool.RejectedH\x00\x42\x07\n\x05\x65vent\"1\n\x05\x41\x64\x64\x65\x64\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\r\n\x05vsize\x18\x02 \x02(\x05\x12\x0b\n\x03\x66\x65\x65\x18\x03 \x02(\x03\"W\n\x07Removed\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\x0e\n\x06reason\x18\x02 \x02(\t\x12\r\n\x05vsize\x18\x03 \x02(\x05\x12\x0b\n\x03\x66\x65\x65\x18\x04 \x02(\x03\x12\x12\n\nentry_time\x18\x05 \x02(\x04\"(\n\x08Rejected\x12\x0c\n\x04txid\x18\x01 \x02(\x0c\x12\x0e\n\x06reason\x18\x02 \x02(\t\"\xba\x01\n\x08Replaced\x12\x15\n\rreplaced_txid\x18\x01 \x02(\x0c\x12\x16\n\x0ereplaced_vsize\x18\x02 \x02(\x05\x12\x14\n\x0creplaced_fee\x18\x03 \x02(\x03\x12\x1b\n\x13replaced_entry_time\x18\x04 \x02(\x04\x12\x18\n\x10replacement_txid\x18\x05 \x02(\x0c\x12\x19\n\x11replacement_vsize\x18\x06 \x02(\x05\x12\x17\n\x0freplacement_fee\x18\x07 \x02(\x03'
+)
```

Comment:
I'm asking as I wouldn't expect e.g. `mempool_pb2.py` (and some of the other files) to change when renaming something unrelated.